department-of-veterans-affairs/abd-vro
# Add BIP keystore and truststore, add BIP evidence api test endpoint.

Add BIP CE test endpoint

## What was the problem?
This PR tries to address two problems.
1. BIP TLS credentials;
2. Add an endpoint to test BIP CE file upload api.

Note: some data has been updated in secret repo. Checkout the secret repo and run scripts/setenv.sh before build.

## How to test this PR
- Step 1. check out secret repo, and build the app followed the normal build procedure.
- Step 2. bring up a app, and test get claim and file upload endpoints in swagger UI.
- Step 3. Set ENV=dev, and ./gradlew :app:dockerComposeDown :app:dcPrune :app:dockerComposeUp
- Step 4. Repeat step 2. the endpoints should respond with error for host not found as your local environment does not have access to BIP API.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
